PIconnect/PIData.py

```diff
@@ -187,6 +195,31 @@ def current_value(self):
         Return the current value of the attribute."""
         return self._current_value()
 
+    def update_value(
+        self,
+        value,
+        ddatetime: Optional[datetime] = None,
+        update_mode: UpdateMode = UpdateMode.NO_REPLACE,
+        buffer_mode: BufferMode = BufferMode.BUFFER_IF_POSSIBLE,
```

Comment:
Could you remove the type hints for now so we remain python 2 compatible for the upcoming release?

Reply:
okies